docker-compose.yaml

```diff
@@ -11,7 +11,7 @@ services:
       - slam-net
     ports:
       - 3000:3000
-    command: ["node", "server.js"]
+    command: ['node', 'server.js']
```

Comment:
Was this copied originally from the API docker-compose? 
This command would make sense for the server, but not the client.

Reply:
Scott clarified in call. 👍